_docs/useful-commands.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Useful Commands
+permalink: "/docs/useful-commands/"
+---
+This page will show you some useful commands when troubleshooting issues with your lancache setup. If this doesn't work out you can always [check out the support page](https://lancache.net/support/).
+
+# Client device commands
+These commands can be run from the client device you are trying to access the lancache from. 
+
+|Command		|  Explanation											|
+|-------------------------------|----------------------------------------|
+|`ipconfig /flushdns`           |   This command will clear all locally stored ip adresses and other DNS records from your device                   |
```

Comment:
Windows only, linux is very system dependant but probably only relevant to windows anyway

Possibly also mention `ipconfig /renew` to update changes to dns via DHCP